kubernetes/pods.go

```diff
@@ -50,7 +50,7 @@ func (a *Adapter) PodInformer(ctx context.Context, endpointChan chan<- []string)
 	queueEndpoints(&podEndpoints, endpointChan)
 
 	// delta triggered updates
-	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+	_, _ = informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
Changed due to golangci-lint:
```bash
golangci-lint run --timeout 5m ./...
kubernetes/pods.go:53:26: Error return value of `informer.AddEventHandler` is not checked (errcheck)
        informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
                                ^
make: *** [Makefile:27: lint] Error 1
```

Reply:
Maybe we should check the error and at least log it for now. Later we can decide to ignore or handle it

Reply:
Sounds good, updated to log the error.